lib/assets/javascripts/cable.coffee

```diff
@@ -5,4 +5,12 @@
   PING_IDENTIFIER: "_ping"
 
   createConsumer: (url) ->
+    if url && (!url.startsWith("wss:") || !url.startsWith("ws:"))
```

Comment:
thanks so much, updating my [tests](https://github.com/kbrock/websocket_url/blob/master/src/websocket_url.spec.js) to add portion to it.